Docker problem: libxml extension failing

Attempted to build docker image on macos


```
    copying src/lxml/isoschematron/resources/xsl/iso-schematron-xslt1/readme.txt -> build/lib.linux-x86_64-2.7/lxml/isoschematron/resources/xsl/iso-schematron-xslt1
    running build_ext
    building 'lxml.etree' extension
    creating build/temp.linux-x86_64-2.7
    creating build/temp.linux-x86_64-2.7/src
    creating build/temp.linux-x86_64-2.7/src/lxml
    x86_64-linux-gnu-gcc -pthread -fno-strict-aliasing -g -O2 -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -I/tmp/pip-build-cWMb1G/lxml/src/lxml/includes -I/usr/local/python/include/python2.7 -c src/lxml/lxml.etree.c -o build/temp.linux-x86_64-2.7/src/lxml/lxml.etree.o -w
    In file included from src/lxml/lxml.etree.c:239:0:
    /tmp/pip-build-cWMb1G/lxml/src/lxml/includes/etree_defs.h:14:31: fatal error: libxml/xmlversion.h: No such file or directory
     #include "libxml/xmlversion.h"
                                   ^
    compilation terminated.
    error: command 'x86_64-linux-gnu-gcc' failed with exit status 1

    ----------------------------------------
Command "/usr/local/python/bin/python -c "import setuptools, tokenize;__file__='/tmp/pip-build-cWMb1G/lxml/setup.py';exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" install --record /tmp/pip-QxYF6g-record/install-record.txt --single-version-externally-managed --compile" failed with error code 1 in /tmp/pip-build-cWMb1G/lxml
The command '/bin/sh -c mod_wsgi-docker-build' returned a non-zero code: 1
```

--- a/.whiskey/action_hooks/pre-build
+++ b/.whiskey/action_hooks/pre-build
@@ -12,3 +12,9 @@ echo "-e ." >> /app/requirements.txt
 
 # Copy application.wsgi into the location mod-wsgi_docker expects it
 cp /app/deploy/application.wsgi /app/ga4gh/
+
+echo " -----> Installing extra libs"
+apt-get update && \
+	apt-get install -y --no-install-recommends \
+	libffi-dev \
+	libxslt1-dev

